src/main/java/screens/WeekViewScreen.java

```diff
@@ -269,6 +271,11 @@ public void windowClosing(WindowEvent e) {
 
         if (confirmed == JOptionPane.YES_OPTION) {
             this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+            try {
```

Comment:
Please remove this. Only the data access classes of our use cases should invoke the save() method of WeekDataAccess.

Reply:
Reviewed an outdated commit.